tests/cases/compiler/newLineFlagWithCR.ts

```diff
@@ -0,0 +1,4 @@
+// @newline: CR
```

Comment:
Sorry about any miscommunication - my understanding is that our harness doesn't support that sort of thing, though it's highly possible that @mhegazy might know something I'm not aware of.